spec/lib/msf/core/auxiliary/nfs_spec.rb

```diff
@@ -0,0 +1,65 @@
+# -*- coding: binary -*-
+
+require 'spec_helper'
+
+RSpec.describe Msf::Auxiliary::Nfs do
+  subject do
+    mod = Msf::Module.new
+    mod.extend(Msf::Auxiliary::Nfs)
+    mod
+  end
+
+  context 'NFS mountability check' do
```

Comment:
Not a blocker:

```suggestion
  describe '#can_mount?` do
```

https://www.betterspecs.org/#describe